docs/design/kademlia_audit_gating.md

```diff
@@ -0,0 +1,94 @@
+# Kademlia Audit Gating
+
+## Abstract
+
+StorageNode B is added to StorageNode A's routing table only if StorageNode A trusts Satellite C and Satellite C as verified StorageNode B has a high enough identity-generate difficulty (CPU proof-of-work) and disk space. If StorageNode B does not have this verification, it is added to StorageNode A's Routing Table Antechamber.
+
+## Background
+
+### White paper: 4.6.1 Mitigating Sybil attacks
+While we’ve adopted the proof-of-work scheme S/Kademlia proposes to partially address Sybil attacks, we extend Kademlia with an application specific integration to further defend our network. 
+
+
+Given two storage nodes, A and B, storage node B is not allowed to enter storage node A’s routing table until storage node B can present a signed message from a Satellite C that storage node A trusts claiming that B has passed enough audits that C trusts it (sections 4.13 and 4.15). This ensures that only nodes with verified disk space have the opportunity to participate in the routing layer.
```

Comment:
Also, is "passed enough audits" definitely the criteria that's measured? Because I see this first goal below: "Add a signed message from Satellites to authenticate whether a Node has a high enough ID generation difficulty and disk space." While these are certain qualifications to meet, I don't think I would describe this evaluation as an audit just because that might lead to confusion with the audit system. Unless I'm misunderstanding and we are also checking for how many successful audits they have on record?

Reply:
That is very true. I think initially we had intended it to be directly based on the audit system, but after further discussion, we decided to make slightly different. This background section is directly taken from the whitepaper.